runtime/js/fs_node.js

```diff
@@ -360,7 +360,7 @@ function MlNodeFd() {}
 //Provides: caml_sys_open_for_node
 //Requires: MlNodeFd
 function caml_sys_open_for_node(fd, flags) {
-  if (flags.name) {
+  if (flags.altname) {
```

Comment:
@hhugo Do you remember what this was for?

Reply:
I think it was related to https://github.com/grain-lang/grain/pull/614

Reply:
trying to read from fd 0 in the node interpreter gives me.
```
fs.readSync(0,b)
Uncaught Error: EAGAIN: resource temporarily unavailable, read
    at Object.readSync (node:fs:742:18) {
  errno: -11,
  code: 'EAGAIN',
  syscall: 'read'
}
````
while reading from '/dev/stdin' works.